lib/src/Uncertainty/Distribution/Triangular.cxx

```diff
@@ -162,8 +162,9 @@ Scalar Triangular::computeCDF(const Point & point) const
 /* Get the characteristic function of the distribution, i.e. phi(u) = E(exp(I*u*X)) */
 Complex Triangular::computeCharacteristicFunction(const Scalar x) const
 {
-  if (std::abs(x) < pdfEpsilon_) return 1.0;
-  if (std::abs(x) < 1.0e-8) return Complex(1.0, (a_ + b_ + m_) * x / 3.0);
+  // Here we need a high order approximation as the exact formula is ill conditioned for small values of x
```

Comment:
A unit test will make this update more robust.

Reply:
Excellent suggestion. Done in t_Triangular_std.cxx and t_Trapezoidal_std.cxx, using Maple reference values (see validation/srd/valid_triangular_cf.mws and validation/srd/valid_trapezoidal_cf.mws)